libs/utils/trace.py

```diff
@@ -394,6 +395,23 @@ def getTasks(self):
         """
         return self._tasks_by_pid.TaskName.to_dict()
 
+    def show(self):
+        """
+        Open the parsed trace using the most appropriate native viewer.
+
+        The native viewer depends on the specified trace format:
+        - ftrace: open using kernelshark
+        - systrace: open using a browser
+
+        In both cases the native viewer is assumed to be available in the host
+        machine.
+        """
+        if isinstance(self.ftrace, trappy.FTrace):
+            return os.popen("kernelshark {}".format(self.ftrace.trace_path))
+        if isinstance(self.ftrace, trappy.SysTrace):
```

Comment:
Is that case possible ? AFAICT we only store a `trappy.FTrace` object in there. It's a bit different for our android workloads (https://github.com/ARM-software/lisa/blob/master/libs/utils/android/workload.py#L146) where this can work.

Reply:
That's not what I see from __parseTrace()
... and, AFAIR, I've already used Trace() in the past on top of a systrace file ;)